apps/samradsgatt/server.ts

```diff
@@ -0,0 +1,11 @@
+import { bootstrap } from '@island.is/infra-next-server'
+import proxyConfig from './proxy.config.json'
+
+bootstrap({
+  name: 'samradsgatt',
+  appDir: 'apps/samradsgatt',
+  proxyConfig,
+  externalEndpointDependencies: process.env.BASE_URL
+    ? [process.env.BASE_URL]
+    : [],
```

Comment:
BASE_URL is not defined in the infra file. I don't think we need to do a DNS health check against the deployed BASE_URL (or why auth-admin-web does it). Probably fine to remove.